CDN/postpreinstall.sh

```diff
@@ -31,13 +31,23 @@ reboot_device()
 	reboot now
 }
 
+install_aria2()
+{
+	echo "Installing aria2"
+	dpkg -i /tmp/aria2_deb/*
+}
+
 post_install()
 {
 echo "Running post install scripts"
 restore_files
+
 # Create a dir for searchdb (if not present)
 mkdir -p /opt/searchEngine/bleveDbDir/
 mkdir -p /home/admin/GoK/
+
+install_aria2
+
 systemctl enable searchserver
```

Comment:
Wouldn't it be auto cleaned at reboot?

Reply:
yes, it will be. But if you do not reboot it won't. We had a case, where the file remained and wget did not overwrite the file (ofcourse there was no reboot). So clean up is safer.

Reply:
Done